great_expectations/datasource/fluent/sources.py

```diff
@@ -269,6 +269,7 @@ def _bind_asset_factory_method_if_not_present(
             def _add_asset_factory(
                 self: Datasource, name: str, **kwargs
             ) -> pydantic.BaseModel:
+                # TODO: identify dc kwargs and pass them to `add_asset`
```

Comment:
@Kilo59 Not sure if this is relevant here, but just wanted to note that differently-purposed `DataConnector` subclasses accept their corresponding (most likely different) "kwargs".  Thanks.

Reply:
I've done this.